CMakeLists.txt

```diff
@@ -420,6 +420,8 @@ if (${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
     set (BARRIER_BUNDLE_APP_DIR ${BARRIER_BUNDLE_DIR}/Barrier.app)
     set (BARRIER_BUNDLE_BINARY_DIR ${BARRIER_BUNDLE_APP_DIR}/Contents/MacOS)
 
+    set (CMAKE_OSX_ARCHITECTURES "arm64;x86_64")
+    
     configure_files (${BARRIER_BUNDLE_SOURCE_DIR} ${BARRIER_BUNDLE_DIR})
```

Comment:
As per my comment in this PR, I think this variable should be disabled for the moment until we can address the Qt issue. Currently Mojave fails to build with this set.cc: @p12tic.

Reply:
This has been disabled in 0642496b, builds on all agents now pass.